devops/openshift/queue-consumers-deploy.yml

```diff
@@ -267,18 +269,18 @@ objects:
       minReplicas: "${{REPLICAS}}"
       maxReplicas: 10
       metrics:
-          - type: Resource
-            resource:
-              name: cpu
-              target:
-                type: Utilization
-                averageUtilization: 80
-          - type: Resource
-            resource:
-              name: memory
-              target:
-                type: AverageValue
-                averageValue: "230M"
+        - type: Resource
+          resource:
+            name: cpu
+            target:
+              type: Utilization
+              averageUtilization: 80
+        - type: Resource
+          resource:
+            name: memory
+            target:
+              type: AverageValue
+              averageValue: "230M"
```

Comment:
Why was it removed?

Reply:
Nothing was removed. It was just re-formatted. 
Btw. I use the YML plugin as we all discussed.

![image](https://github.com/bcgov/SIMS/assets/54600590/96515c31-a8ae-48e1-962b-ea0c5673bd30)

![image](https://github.com/bcgov/SIMS/assets/54600590/68e6322b-3558-4278-b7d5-f0455b38d9fc)

Reply:
yes, I quickly checked and thought it was removed. OK. Thank you!